frontend/pages/hosts/components/DiskSpaceIndicator/DiskSpaceIndicator.tsx

```diff
@@ -20,6 +21,14 @@ const DiskSpaceIndicator = ({
   platform,
   tooltipPosition = "top",
 }: IDiskSpaceIndicatorProps): JSX.Element => {
+  // Check if storage measurement is not supported (sentinel value -1)
+  if (
+    typeof gigsDiskSpaceAvailable === "number" &&
```

Comment:
Could be nice with a single test validating this case on the frontend, for this file and HostSummary